File: lib/csharp/tests/VABotTest/ChildBot/Models/SkillState.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

using Luis;

namespace ChildBot.Models
{
    public class SkillState
    {
        public string Token { get; set; }

        public SkillBotLuis BotLuisResult { get; set; }

        public void Clear()
        {
        }
    }
}

File: templates/typescript/samples/sample-skill/src/models/skillState.ts
/**
 * Copyright(c) Microsoft Corporation.All rights reserved.
 * Licensed under the MIT License.
 */

import { injectable } from 'inversify';

@injectable()
export class SkillState {
    public token = '';
    public timeZone: Date = new Date();
    // eslint-disable-next-line @typescript-eslint/no-empty-function
    public clear(): void {}
}
